python-lib/gluonts_forecasts/model_handler.py

```diff
@@ -84,11 +86,28 @@
         ESTIMATOR: TemporalFusionTransformerEstimator,
         TRAINER: Trainer
     },
+    "npts": {
+        LABEL: "NPTS",
+        CAN_USE_EXTERNAL_FEATURES: False,
+        ESTIMATOR: None,
+        PREDICTOR: NPTSPredictor,
+        TRAINER: None,
+        CAN_USE_CONTEXT_LENGTH: False,
+    },
+    "arima": {
```

Comment:
```suggestion
    "autoarima": {
```

Reply:
Careful to change the key in recipe.json too if you change this one